dmutils/content_loader.py

```diff
@@ -90,7 +217,10 @@ def __init__(self, manifest, content_directory):
         ]
 
     def get_question(self, question):
-        return self._questions.get(question, {}).copy()
+        q = self._questions.get(question, {}).copy()
+        if q:
+            q['id'] = question
```

Comment:
I think this may have been an example of the test data not matching the real data. I have since added `id`s to the test data so I think this can be removed.